src/lib/select/select.spec.ts

```diff
@@ -1362,7 +1369,8 @@ describe('MdSelect', () => {
   selector: 'basic-select',
   template: `
     <div [style.height.px]="heightAbove"></div>
-    <md-select placeholder="Food" [formControl]="control" [required]="isRequired">
+    <md-select placeholder="Food" [formControl]="control" [required]="isRequired"
+      [tabIndex]="tabIndexOverride">
```

Comment:
Chatted with Jeremy about this, and it makes sense to support both. Since the HTML property is `tabIndex`, you should bind to `tabIndex`. But people should still be able to use `tabindex="5"` without binding the way they normally would. We can achieve this with another @Attribute binding.